dev/NavigationView/NavigationView.cpp

```diff
@@ -584,6 +588,16 @@ void NavigationView::UpdateAdaptiveLayout(double width, bool forceSetDisplayMode
     {
         MUX_FAIL_FAST();
     }
+    if (!forceSetDisplayMode && m_InitialNonForcedModeUpdate) {
+        if (displayMode == winrt::NavigationViewDisplayMode::Minimal && PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Auto) {
+            ClosePane();
+        }
+        m_InitialNonForcedModeUpdate = false;
+    }
+
+    if (forceSetDisplayMode) {
+        m_InitialNonForcedModeUpdate = true;
+    }
```

Comment:
This part maybe introduce unexpected closepane and I will need more test

Reply:
My understanding is that forceSetDisplayMode will only be true if it is the first call of SetDisplayMode as indicated by [this comment](https://github.com/chingucoding/microsoft-ui-xaml/blob/4ec745e0485ab57b49027417bb6dbacdc395a0b3/dev/NavigationView/NavigationView.cpp#L540).

Regarding the tests, what behaviour would you need me to check?😅
The only thing the current fix does is close the pane if NavView is in auto mode and because of it's width it is set to DisplayMode=Minimal.

Reply:
Forgot to mention that I already added a test for this 
> The only thing the current fix does is close the pane if NavView is in auto mode and because of it's width it is set to DisplayMode=Minimal.

behaviour. Sorry for the confusion.

Reply:
I go through all parts of forceSetDisplayMode, and I can't say you are wrong.

Since you are only fix the initialization problem, will below solution works for you?
Remove lines from 589 to 598 and set m_InitialNonForcedModeUpdate = true in NavigationView.h?
and then replace the code with
 ```
    if (!forceSetDisplayMode && m_InitialNonForcedModeUpdate) { 
        if (displayMode == winrt::NavigationViewDisplayMode::Minimal && PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::Auto) { 
            ClosePane(); 
            m_InitialNonForcedModeUpdate = false; 
        } 
    }
```

Reply:
or possible just remove lines from 596 to 598

Reply:
Yes that will work and is a lot more elegant. I will fix this now. 
Thank you so much for the suggestions!